src/AccessibilityInsights.SharedUx/Dialogs/LoadingBox.xaml

```diff
@@ -1,23 +0,0 @@
-<!-- Copyright (c) Microsoft. All rights reserved.
```

Comment:
`LoadingBox` contained a reference to `ProgressRingControl`, but is not in use. I thought it would be better to remove the class than update it.